src/coneslam/imgproc_test.cc

```diff
@@ -19,7 +24,7 @@ int main() {
     float gyroz;
     memcpy(&gyroz, framebuf+26+8, 4);
     printf("%d: gyroz=%f ", frame, gyroz);
-    int ncones = coneslam::FindCones(framebuf+55, gyroz, 10, xbuf, thetabuf);
+    int ncones = coneslam::FindCones(/*yuvimg=*/framebuf+55, /*thresh=*/config.cone_thresh, /*gyroz=*/gyroz, /*nout=*/10, /*x_out*/xbuf, /*bearing_out=*/thetabuf);
```

Comment:
This is Clang syntax, which enables a clang-tidy check which warns if the signature of the called function changed.